clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -161,16 +161,27 @@ Improvements to clang-tidy
 New checks
 ^^^^^^^^^^
 
+- New :doc:`bugprone-derived-method-shadowing-base-method
+  <clang-tidy/checks/bugprone/derived-method-shadowing-base-method>` check.
+
+  Finds derived class methods that shadow a (non-virtual) base class method.
+ 
+ - New :doc:`bugprone-derived-method-shadowing-base-method
```

Comment:
Duplicated entry. Incorrect merge/rebase?

Reply:
Augh, yeah, bad rebase.  I've re-pushed and I believe it's deduplicated now.